submissions/nadsatt/friends-app/modules/icon-item.js

```diff
@@ -0,0 +1,37 @@
+import svgs from './svgs.js';
+
+export class Icon {
```

Comment:
Please replace it with `FilterButton` and make it as dependency of `Filter` or `Sorter`